Makefile

```diff
@@ -10,6 +10,9 @@ IMGNAME?=kubernetes-ingress-controller
 IMAGE = $(REGISTRY)/$(IMGNAME)
 IMG ?= controller:latest
 NCPU ?= $(shell getconf _NPROCESSORS_ONLN)
+export ENTERPRISE_REPO = kong/kong-gateway
+export ENTERPRISE_TAG = 2.5.0.0-alpine
```

Comment:
IMO grab these values from environment variables. Matrix the test in the CI setup to validate kic against stable `kong/kong-gateway:latest` as well as against unstable / what's coming up `kong/kong-gateway-internal:2.5-nightly`

Reply:
seems I meant to say `kong/kong-gateway-internal:master-nightly`

Reply:
I think this is a great idea, do you mind create an issue on this ? We would like to bring this to the team and have some discussion.

The main issue this PR resolves is to get KIC gated by enterprise release version (stable) 2.5.

Reply:
on behalf of you, come up with the issue https://github.com/Kong/kubernetes-ingress-controller/issues/1844. Please go ahead, and comment, put more details there.